docker-compose.yml

```diff
@@ -234,6 +234,13 @@ services:
     restart: unless-stopped
     tty: false
 
+  explorer:
+    container_name:  hedera-explorer
+    image: "glazarov95/hedera-explorer:latest"
```

Comment:
Q: is this temporary? Should be from an official location in gcp or github no?

Reply:
Yes, this is temporary. Can we upload the image to official gcr, so I can change the path to official source ?

Reply:
This will be handled by release engineering as part of our CI and release process. This is currently on the release engineering schedule for sometime this quarter. This is also closely coupled with automating our image builds for the network node and automating version updates for the mirror node software.